drift/test/manager/manager_selectable_test.dart

```diff
@@ -0,0 +1,217 @@
+// ignore_for_file: unused_local_variable
+
+import 'package:drift/drift.dart';
+import 'package:test/test.dart';
+
+import '../generated/todos.dart';
+import '../test_utils/test_utils.dart';
+
+/// All managers `Managers` are Selectable classes that
+/// are used by the Manager API to return results.
+/// This test will ensure that they all behave as expected, no matter what filters/ordering/limit/offsets/references are applied.
+
+void main() {
+  late TodoDb db;
+
+  setUp(() {
+    db = TodoDb(testInMemoryDatabase());
+  });
+
+  tearDown(() => db.close());
+
+  test('manager - selectable tests', () async {
+    final stores = [
```

Comment:
Can we simplify the initialization here somewhat (perhaps by using non-manager batches or iterating over arrays)? That code makes up like 80% of this test.

Reply:
Done